src/Tabs/StyledTab.js

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import Flex from '../Flex';
+
+const activeStyles = ({ active, theme }) => {
+  const color = theme.colors.intent.brand.dark;
+
+  return (
+    active &&
+    css`
+      color: ${color};
+      border-color: ${color};
+
+      &:focus,
+      &:hover {
+        background: transparent;
+      }
+    `
+  );
+};
+
+const Tab = styled(Flex)`
+  cursor: pointer;
+  outline: none;
+
+  &:focus,
+  &:hover {
+    background: ${props => props.theme.colors.background.muted};
+  }
+
+  ${activeStyles};
+`;
+
+Tab.propTypes = {
+  active: PropTypes.bool,
+  tabIndex: PropTypes.string
```

Comment:
Do we care about the `tabIndex` within the `StyledTab`?

Reply:
We do to attribute focus styles. The tab should be focusable and its not a focusable element by default.